src/components/ColumnTitle/ColumnFilter.tsx

```diff
@@ -89,8 +104,10 @@ export const ColumnFilter: FunctionComponent<ColumnFilterProps> = (props) => {
     return <Popover
         trigger="click"
         content={content}
+        visible={visible}
```

Comment:
You can use [onVisibleChange ](https://ant.design/components/tooltip/#API) property of `<Popover />` to handle the situation when it's not closed on outside click

Reply:
Fixed